doc/classes/Basis.xml

```diff
@@ -78,6 +78,7 @@
 			<return type="Quat">
 			</return>
 			<description>
+				Assuming that the matrix is a proper rotation matrix (orthonormal matrix with determinant +1), return a quaternion. See [method get_euler] if you need Euler angles instead.
```

Comment:
It seems that this method is self-orthonormalizing, so no assumptions are made. https://github.com/godotengine/godot/blob/master/core/math/basis.cpp#L370 (also I noticed the comments of these methods mention Euler angles when they shouldn't, but this can be fixed later if you don't want to)

Also, is it a good idea to point users towards `get_euler`? I think if users are specifically looking for Euler angles they will Ctrl+F for it, but in general Euler angles should be discouraged in my opinion.